tests/test_builders/test_app_builder.py

```diff
@@ -35,6 +35,7 @@
     "json_file, return_type",
     [
         pytest.param("tests/utils/manifest_files/manifesttest.json", ApplicationView),
+        pytest.param("tests/utils/manifest_files/no_apps_manifest.json", ApplicationView),
```

Comment:
Do we need to parametrize the return_type argument w/`ApplicationView` if it's the same in both test cases?